presto-docs/src/main/sphinx/release/release-341.md

```diff
@@ -0,0 +1,163 @@
+# Release 341 (XX Aug 2020)
+
+## General Changes
+
+* Add support for variable-precision `TIME` type. ({issue}`4381`)
+* Add support for variable precision `TIME WITH TIME ZONE` type. ({issue}`4905`)
+* Add Iceberg connector.
+* Add {func}`human_readable_seconds` function. ({issue}`4344`)
+* Add [`reverse`](function_reverse_varbinary) function for `VARBINARY`. ({issue}`4741`)
```

Comment:
`reverse` is rendered differently to to other function links for some reason:
![image](https://user-images.githubusercontent.com/890111/92527408-3c726e00-f227-11ea-973d-6a393a77b4ae.png)

Reply:
Yes, that's because I had to link to it as a regular reference. There are multiple versions of `reverse` and a `{func}` link ends up pointing at the wrong one.